Makefile.am

```diff
@@ -43,6 +43,18 @@ package-clean:
 	rm -f package/debian.*
 	rm -f package/*.dsc*
 
+if ENABLE_COVERAGE
+# Run `make coverage` after all the tests have been run.
+# See also make -f Makefile.repo coveralls
+COVERAGE_INFO = coverage/coverage.info
+.PHONY: coverage
+coverage:
+	mkdir -p $(dir $(COVERAGE_INFO))
+	lcov --capture --no-external --exclude '*/testsuite*/*' --output-file $(COVERAGE_INFO) --directory .  --quiet
+	lcov --list $(COVERAGE_INFO)
```

Comment:
NP: maybe I'd switch the `lcov --list` and the `genhtml` commands. It would be nice to see the coverage numbers after running `make coverage`, in this order it could scroll up...

Reply:
genhtml produces no output in the build log :)